pyanaconda/core/util.py

```diff
@@ -666,19 +666,33 @@ def service_running(service):
     return ret == 0
 
 
-def enable_service(service):
-    """ Enable a systemd service in the sysroot """
-    ret = _run_systemctl("enable", service, root=getSysroot())
+def enable_service(service, root=None):
+    """ Enable a systemd service in the sysroot.
+
+    :param str: name of the service to enable
+    :param: path to the sysroot or None to use default sysroot path
```

Comment:
Please remove double space before `  path...`

Reply:
Fixed up the function headers.